doc/md/declarative/declarative-hcl.mdx

```diff
@@ -0,0 +1,221 @@
+---
+id: declarative-hcl-singleschema
+title: Getting started with HCL declarative migrations for a MySQL schema
+slug: /declarative/declarative-hcl
+---
+
+import InstallationInstructions from '../components/_installation_instructions.mdx'
+
+### Introduction
+
+Atlas makes it easy to manage your database using the declarative migrations workflow. Declarative migrations focus on describing the desired state of the database and allow the migration engine to plan and execute the necessary actions to reach that state.
+
+In this quickstart guide, we will go through the three essential steps for using Atlas to manage your database schema: inspecting your database, authoring the desired changes, and applying those changes.
+
+### Installing Atlas
+
+<InstallationInstructions />
+
+### Warming up
+To demonstrate the examples in the article, let's create a docker container with an `example` database and insert a table in it.
+Use the following command to run a MySQL Docker container
```

Comment:
Where is the command?

Reply:
Take it from here:
https://atlasgo.io/getting-started#start-a-local-database-container